nf20 readme update (#86)
blues/app-accelerators: 20-motor-monitor-and-vibration/README.md

* (optional - for power outage events) [LiPo battery with JST cable](https://shop.blues.io/products/5-000-mah-lipo-battery)
* [Notecarrier-B](https://shop.blues.io/products/carr-b)
* [Notecard](https://blues.io/products/notecard/)
* A switched motor that has a 3v3 input or output that controls or indicates activity. (See the next section for details.)
* A pair of jumper cables

## Motor Choice and Testing

> This project illustrates an app that monitors an industrial motor for activity and vibration and can in principle operate with almost any kind of motor. In an industrial setting this would typically be a an electric motor already installed at a facility. In addition to the motor, a 3.3-5v logic signal is used to indicate when the motor is active. In cases where this is not already present, it can be retrofitted using an optocoupler placed in parallel with the motor's electrical supply on the input side, which switches a 3.3v line from the Notecarrier on the output side, indicating when the motor is active.

Reviewer: ```suggestion
This project illustrates an app that monitors an industrial motor for activity and vibration, and can in principle operate with almost any kind of motor. In an industrial setting this would typically be an electric motor already installed at a facility. In addition to the motor, a 3.3-5v logic signal is used to indicate when the motor is active. In cases where this is not already present, it can be retrofitted using an optocoupler placed in parallel with the motor's electrical supply on the input side, which switches a 3.3v line from the Notecarrier on the output side, indicating when the motor is active.
```

During initial testing and evaluation, you probably don't have an industrial motor sitting on your desk! Fortunately, the project can be used with something smaller, such as a [mains-powered desk fan](https://www.amazon.com/AmazonBasics-Speed-Air-Circulator-7-Inch/dp/B082MY2MX3/ref=sr_1_7). For the motor activity signal, you could use the optocoupler as outlined above. Alternatively, you may simply use an [IoT relay](https://www.digital-loggers.com/iot2.html) to control the motor activity from the Notecard.

## Setup overview

> With the motor and activity line already setup and ready, you will first connect the Notecarrier to a computer to configure the Notecard. Once that is complete, the Notecarrier is disconnected from the computer and is securely mounted onto the motor. The motor activity signal is connected to the Notecarrier from the motor and the the Notecarrier is supplied via a USB power brick.

Reviewer: ```suggestion
With the motor and activity line already set up and ready, you will first connect the Notecarrier to a computer to configure the Notecard. Once that is complete, the Notecarrier is disconnected from the computer and is securely mounted onto the motor. The motor activity signal is connected to the Notecarrier from the motor and the Notecarrier is supplied via a USB power brick.
```

* how often motion events are captured on the Notecard
* how often motion events are sent to Notehub

### Notecard Configuration Complete

That completes the Notecard configuration. Now you can disconnect the Notecarrier from the computer's USB port and power it from the USB power brick and mount the Notecarrier and Notecard to the motor.

## Create a Route with JSONata

> The app uses a JSONata transform in a Notehub route to determine the amount of vibration and to signal an alert when vibration is out of range for the current state of the motor.

Reviewer: ```suggestion
The app uses a JSONata transform in a Notehub route to determine the amount of vibration, and to signal an alert when vibration is out of range for the current state of the motor.
```

1. Create a new route (such as an HTTP route) with notefile `_motion.qo`. Copy the contents of the `route.jsonata` from this project into the JSONata transformation text input in the route setup page.

2. Configure the remaining route details. For HTTP routes, this includes setting the URL of the endpoint that routed events are delivered to. (For testing, I used an http route to `https://website.site` to show the result of the route. You can use any route that works for your use case.)

For more details Notehub and routes, see [Routing Data with Notehub](https://dev.blues.io/notehub/notehub-walkthrough/#routing-data-with-notehub).

## Configure Vibration Alerts in Notehub

> Vibration alerts are sent when the detected vibration is out of range compared to what is expected when motor is active or not.

Reviewer: ```suggestion
Vibration alerts are sent when the detected vibration is out of range compared to what is expected when the motor is active or not.
```
